Text to describe set up that may be necessary to use the Emacs text editor (#25)
carpentries-incubator/intro-to-modern-fortran: learners/setup.md

:::::::::::::::::::::::::

## Text Editors and Integrated Development Environments (IDEs)

> While Fortran files are plain text can could be edited with any plain text editor, it is recommened that you use an editor that is capable of _syntax highlighting_. There are a number of these available.

Reviewer: ```suggestion
While Fortran files are plain text and can be edited with any plain text editor, it is recommened that you use an editor that is capable of _syntax highlighting_. Setup instructions for some editors are available below.
```

### Emacs

> [Emacs][emacs-link] is a cross-platform text editor available for Linux, MacOS, and Windows. It will contain syntax highlighting for many Fortran file, but additional file extensions, such as `.X90`  may be valid Fortran files but will not be picked up in Emacs _f90-mode_. This mode allows for tab-completion of Fortran statements and the addition of a drop-down menu that provides additional Fortran-related options.

Reviewer: ```suggestion
[Emacs][emacs-link] is a cross-platform text editor available for Linux, MacOS, and Windows. Emacs applies syntax highlighting for most Fortran files, but additional file extensions, such as `.X90`  may be valid Fortran files but will not be picked up in Emacs _f90-mode_. This mode allows for tab-completion of Fortran statements and the addition of a drop-down menu that provides additional Fortran-related options.
```

Author: Many thanks - I've committed the suggestion
